### ISSUE TITLE
Electron modularization for other back-end languages

### ISSUE THREAD

**Issue by up-to-you** [electron/electron#10791]
Is there a way to build (and package) electron rendering engine as standalone executable binary for further interaction with other programming languages as a child process ? For example to exclude nodejs libraries? I'm aware about libchromiumcontent, but it seems to be reinventing the wheel, because electron app team already done most of the job.

**Comment by MarshallOfSound** [comment]
> Is there a way to build (and package) electron rendering engine

Electrons rendering engine is Chromium, `libchromiumcontent` is how we prebuild most of Chromium for Electron itself to use.  There isn't a way to easily pull out the Electron core logic to another language API as most if not all of Electrons core API's rely on NodeJS natives to make them work (v8 converters, nan bindings, etc.)

TLDR: We can't easily expose ways to bind Electron to a non-nodejs thing

If you want to use Electron from another language you can just spawn Electron executables and give them JS files to run.
